src/market/market.py

```diff
@@ -152,27 +72,27 @@ def get_mint_decimals(conn: Client, mint_pub_key: PublicKey) -> int:
         return MINT_LAYOUT.parse(bytes_data).decimals
 
     def bids_address(self) -> PublicKey:
-        return PublicKey(self._decode.bids)
+        return self.state.bids
 
     def asks_address(self) -> PublicKey:
-        return PublicKey(self._decode.asks)
+        return self.state.asks
```

Comment:
Can we move these to market_state.py

Reply:
deleted actually. we can just access the field name if needed.